addons/sale_management/views/sale_order_template_views.xml

```diff
@@ -55,7 +55,10 @@
                     </group>
                     <notebook name="main_book">
                         <page string="Lines" name="order_lines">
-                        <field name="sale_order_template_line_ids" widget="section_and_note_one2many">
+                        <field
+                            name="sale_order_template_line_ids"
+                            widget="section_and_note_one2many"
+                        >
```

Comment:
Unrelated diff

Reply:
Yes, I forgot to revert this (I initially intended to use the combo widget here as well before simply disallowing combo products in sale order templates).